src/mapping-v3-core.ts

```diff
@@ -1047,7 +1309,7 @@ function refreshContract<T>(contract: T, timestamp: BigInt): Contract | null {
     contractEntity.curationRegistry = contract.artblocksCurationRegistryAddress();
     // flagship never auto approves artist split proposals for all changes
     contractEntity.autoApproveArtistSplitProposals = false;
-  } else if (contract instanceof GenArt721CoreV3_Engine) {
+  } else if (isEngineOrEngineFlex) {
```

Comment:
nit - WDYT about a comment somewhere in this code block stating that all flex-specific functionality is not refreshed, due to the flex-specific handlers updating relevant entities on their own?